docs/Sonarr/Sonarr-Release-Profile-RegEx.md

```diff
@@ -176,17 +98,7 @@ Add this to your `Preferred (3)` with a score of [75]
 
 ## Second Release Profile
 
-<!-- [trash_id: 37cf8cdd57c8fb4a8b68f36e00e40de2] -->
-!!! warning
```

Comment:
This changes. The warning is now below the `### Golden rule` header.

Reply:
This should be right after the second profile, else people think it's only for this regex and not everything followed in the second release profile